l10n_ch_import_cresus/wizard/l10n_ch_import_cresus.py

```diff
@@ -0,0 +1,344 @@
+        return (state, msg)
+    
+    @api.multi
+    def _standardise_data(self, data):
+        """ This function split one line of the CSV into multiple lines.
+        Cresus just write one line per move, 
+        
+
+        """
+        new_openerp_data = []
+        tax_obj = self.env['account.tax']
+        user_obj = self.env['res.users']
+        account_obj = self.env['account.account']
+        cp = user_obj.browse([self.env.uid]).company_id
+        company_partner = cp.partner_id.name
+        standard_dict = dict(izip_longest(self.HEAD_OPENERP, []))
+        previous_date = False
+        for line_cresus in data:
+            is_negative = False
+            current_date_french_format = datetime.strptime(line_cresus['date'],
+                                                           '%d.%m.%Y')
+            current_openerp_date = datetime.strftime(
+                current_date_french_format,
+                DEFAULT_SERVER_DATE_FORMAT)
+            default_value = standard_dict.copy()
+            if (not previous_date) or previous_date != current_openerp_date:
+                default_value['date'] = current_openerp_date
+                default_value['ref'] = line_cresus['pce']
+                default_value['journal_id'] = self.journal_id.name
+                default_value['period_id'] = self.period_id.code
+                previous_date = current_openerp_date
+            else:
+                default_value['date'] = None
+                default_value['ref'] = None
+                default_value['journal_id'] = None
+                default_value['period_id'] = None
+            decimal_amount = float(
+                line_cresus['amount'].replace('\'', '').replace(' ', ''))
+            if decimal_amount < 0:
+                default_value['line_id/credit'] = abs(decimal_amount)
+                default_value['line_id/debit'] = 0.0
+                default_value['line_id/account_id'] = line_cresus['debit']
+                is_negative = True
+            else:
+                default_value['line_id/debit'] = abs(decimal_amount)
+                default_value['line_id/credit'] = 0.0
+                default_value['line_id/account_id'] = line_cresus['debit']
+            tax_code = None
+            analytic_code = None
+            tax_code_inverted = None
+            tax_current = None
+            analytic_code_inverted = None
+            if line_cresus['typtvat']:
+
+                tax_ids = tax_obj.search([('tax_cresus_mapping',
+                                                    '=',
+                                                    line_cresus['typtvat']),
+                                                   ('price_include',
+                                                    '=',
+                                                    True)],
+                                         limit=1)
+                if tax_ids:
+                    tax_current = tax_obj.read(tax_ids, ['name'])[0]
```

Comment:
If you want a list of tax name, you can write it:

    taxes.mapped('name')

(using taxes as it is a recod set)